crates/uv/tests/lock.rs

```diff
@@ -1465,7 +1465,7 @@ fn lock_git_sha() -> Result<()> {
         [[distribution]]
         name = "uv-public-pypackage"
         version = "0.1.0"
-        source = { git = "https://github.com/astral-test/uv-public-pypackage?rev=main#0dacfd662c64cb4ceb16e6cf65a157a8b715b979" }
+        source = { git = "https://github.com/astral-test/uv-public-pypackage", precise = "b270df1a2fb5d012294e9aaf05e7e0bab1e6a389", revision = "main" }
```

Comment:
Maybe we use `commit` instead of `precise` for this, since it's somewhat user-facing?

Reply:
Yeah I like it. Done.